sdk/communication/Azure.Communication.PhoneNumbers/tests/PhoneNumberAdministrationClient/PhoneNumberAdministrationClientLiveTests.cs

```diff
@@ -247,7 +249,8 @@ public async Task CreateReservationErrorState(string? locale)
         [Test]
         [TestCase(null)]
         [TestCase("en-US")]
-        public async Task CreateReservation(string? locale)
+        [Ignore("TollFreeNumbersAreDisabledTemporarily")]
```

Comment:
We are going to replace these anyways. I disabled them as they are disabled for now and are breaking the tests